R/system.R

```diff
@@ -2953,3 +2953,21 @@ filter_cascade <- function(.data, ...){
   }
   df
 }
+
+#
+#'@export
```

Comment:
Add some comment to describe this function.

Reply:
Added comment.